### ISSUE TITLE
Error "Missing attendee email" when trying to create an event in google calendar

### ISSUE THREAD

**Issue by SuperKa3a** [calcom/docker#192]
### Discussed in https://github.com/calcom/docker/discussions/190

<div type='discussions-op-text'>

<sup>Originally posted by **SuperKa3a** December 20, 2022</sup>
Hi!
I installed cal.com via docker.
Then I set up the Google Calendar integration as written here: https://github.com/calcom/cal.com#obtaining-the-google-api-credentials
After that, the application appeared in the list of all applications and I added it.
Now when I try to book an appointment, I get an error in the logs: "There was an error contacting google calendar service:  GaxiosError: Missing attendee email."
I took the request being sent to the google api, compared it with what is expected and realized that when sending, an extra "rest" nesting is generated in the "attendees" block: 
https://pastebin.com/raw/5TESLGnh

When i repeated the given request in postman i got the same error, but when i removed the "rest" block, the request became successful and the event was successfully added to the calendar.

Tell me, has anyone experienced this? What can be done in this case?</div>

**Comment by krumware** [comment]
In the .env that you provide for docker, you may need to remove any quotation marks that wrap the variable value.

Otherwise, this is likely an issue that needs to be opened in the cal.com repository

**Comment by SuperKa3a** [comment]
@krumware do you mean the GOOGLE_API_CREDENTIALS parameter? It exactly matches the contents of the client_secret.json file from the Google application, there are no extra quotes. At the same time, the integration itself was successful, that is, the request and granting of rights

**Comment by krumware** [comment]
Yes, along with other variables. Env config issues is a common cause, so it's helpful to start there. If those are correct (it sounds like they are) then that error should not be specific to docker, but to the cal.com core project.
I haven't heard of that issue otherwise. Have you searched in the /calcom/cal.com issues?

**Comment by SuperKa3a** [comment]
Yes, I searched but didn't find

**Comment by krumware** [comment]
@SuperKa3a were you able to resolve this? I'm curious if updating the image you are using helps this issue

**Comment by libeanim** [comment]
Is this maybe related to [#5796 on calcom/cal.com](https://github.com/calcom/cal.com/issues/5796)?

**Comment by krumware** [comment]
Closing for cleanup, please reopen if still experiencing this issue. Thanks!
